File: backend/app/schemas/enums.py
from enum import StrEnum


class PrimaryGoal(StrEnum):
    fitness = "fitness"
    career = "career"
    discipline = "discipline"
    healing = "healing"
    confidence = "confidence"
    study = "study"


class Tone(StrEnum):
    calm = "calm"
    intense = "intense"
    spiritual = "spiritual"
    practical = "practical"
    stoic = "stoic"


class QuoteLength(StrEnum):
    short = "short"
    medium = "medium"


class Category(StrEnum):
    discipline = "discipline"
    healing = "healing"
    fitness = "fitness"
    career = "career"
    grief = "grief"
    study = "study"
    loneliness = "loneliness"
    self_respect = "self-respect"
    resilience = "resilience"

File: mobile/src/constants/enums.ts
export const PRIMARY_GOALS = ['fitness', 'career', 'discipline', 'healing', 'confidence', 'study'] as const;
export const TONES = ['calm', 'intense', 'spiritual', 'practical', 'stoic'] as const;
export const QUOTE_LENGTHS = ['short', 'medium'] as const;
export const CATEGORIES = ['discipline', 'healing', 'fitness', 'career', 'grief', 'study', 'loneliness', 'self-respect', 'resilience'] as const;

export type PrimaryGoal = (typeof PRIMARY_GOALS)[number];
export type Tone = (typeof TONES)[number];
export type QuoteLength = (typeof QUOTE_LENGTHS)[number];
export type Category = (typeof CATEGORIES)[number];
